receiver/mongodbatlasreceiver/internal/mongodb_atlas_client.go

```diff
@@ -553,3 +555,33 @@ func (s *MongoDBAtlasClient) processDiskMeasurementsPage(
 	}
 	return measurements.Measurements, hasNext(measurements.Links), nil
 }
+
+func (s *MongoDBAtlasClient) GetLogs(ctx context.Context, groupID, hostname, logName string, col_interval time.Duration) (*bytes.Buffer, error) {
+	buf := bytes.NewBuffer([]byte{})
+
+	t := strconv.Itoa(int(time.Now().Add(-col_interval).Unix()))
+	end := strconv.Itoa(int(time.Now().Unix()))
+
+	resp, err := s.client.Logs.Get(ctx, groupID, hostname, logName, buf, &mongodbatlas.DateRangetOptions{StartDate: t, EndDate: end})
+	// resp, err := s.client.Logs.Get(ctx, groupID, hostname, logName, buf, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		return nil, fmt.Errorf("received status code: %d", resp.StatusCode)
+	}
+
+	return buf, nil
+}
+
+func (s *MongoDBAtlasClient) GetClusters(ctx context.Context, groupID string, include, exclude []string) ([]mongodbatlas.Cluster, error) {
```

Comment:
parameters include/exclude don't seem to be used at all here; We should remove these parameters, filtering using these is done elsewhere.